docs/source/transactions.md

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
This file should be moved into the directory `docs/source/endpoints`.

Adjust the paths:

`../../src/plone/restapi/tests/http-examples/`

becomes:

`../../../src/plone/restapi/tests/http-examples/`

Update the `toctrees` in both `docs/source/endpoints/index.md` and `docs/source/index.md`.